allensdk/brain_observatory/behavior/criteria.py

```diff
@@ -0,0 +1,216 @@
+"""
+Functions for calculating mtrain state transitions.
+If criteria are met, return true. Otherwise, return false.
+"""
+
+import logging
+from allensdk.internal.core.exceptions import DataFrameKeyError, DataFrameIndexError
+
+
+logger = logging.getLogger(__name__)
+
+
+def two_out_of_three_aint_bad(session_summary):
```

Comment:
May want to consider adding type annotations? e.g.
``` 
def two_out_of_three_aint_bad(session_summary: pd.DataFrame) -> bool:
```

Reply:
As discussed, it seems that downstream processes that would rely on this code still use legacy python, so I don't want to put in type annotations. I think the docstrings are verbose enough.